packages/@aws-cdk/aws-kinesisfirehose-destinations/test/integ.s3-bucket.lit.expected.json

```diff
@@ -578,9 +578,40 @@
             {
               "Action": "lambda:InvokeFunction",
               "Effect": "Allow",
+              "Resource": [
+                {
+                  "Fn::GetAtt": [
+                    "DataProcessorFunctionAD472B9A",
+                    "Arn"
+                  ]
+                },
+                {
+                  "Fn::Join": [
+                    "",
+                    [
+                      {
+                        "Fn::GetAtt": [
+                          "DataProcessorFunctionAD472B9A",
+                          "Arn"
+                        ]
+                      },
+                      ":*"
+                    ]
+                  ]
+                }
+              ]
+            },
+            {
+              "Action": [
+                "kms:Decrypt",
+                "kms:Encrypt",
+                "kms:ReEncrypt*",
+                "kms:GenerateDataKey*"
+              ],
+              "Effect": "Allow",
               "Resource": {
                 "Fn::GetAtt": [
-                  "DataProcessorFunctionAD472B9A",
+                  "Key961B73FD",
                   "Arn"
                 ]
```

Comment:
This is weird. This permission looks to be duplicated now?

Reply:
Sigh.... I updated them again, and now the assets have changed. I guess the source of truth is what the PR build thinks the integ test should be, so i will work towards getting a successful build.